setup_root.md

```diff
@@ -1,65 +1,211 @@
+```
+$ zendro dockerize -u -p
+```
+
+This command will create docker containers for each Zendro component:
+* [Keycloak](https://github.com/Zendro-dev/Zendro-dev.github.io/blob/documentation-vb/oauth.md): manage users and roles
+* [Single Page App (SPA)](https://github.com/Zendro-dev/single-page-app): graphical interface to send CRUD requests to a Zendro GraphQL endpoint
+* [API](https://github.com/Zendro-dev/graphql-server): CRUD API that can be accessed through a GraphQL query language
+* [API with authenthication](https://github.com/Zendro-dev/graphiql-auth): An implementation of the GraphQL IDE with Zendro login
+
+You can check docker containers by:
+```
+$ docker ps
 ```
-It is also possible to [run the zendro services locally](https://zendro-dev.github.io/zendro_cli.html#start-zendro-service), however you might run into unexpected incompatibilities depending on your local system.
+
+You can check docker logs by:
 ```
-zendro start
+$ docker logs -f <container name>
 ```
 
-### Step 5 (optional): Customize your project
+> ***Please wait until logs indicate the app is running on XXXX port to access Zendro services.***
+
+In default config, the running containers will be on ports:
+
+* Keycloak: http://10.5.0.11:8081
+   * The default keycloak username is *admin* and the password is *admin*.
+
+  ![Keycloak example](figures/kc1.png)
+  ![Keycloak example](figures/kc2.png)
+
+* SPA: http://localhost:8080
+    * The default zendro username is *zendro-admin* and the password is *admin*.
+
+  ![spa example](figures/login.png)
+  ![spa example](figures/spa.png)
+
+* GraphQL API: http://localhost:3000/graphql
+
+  ![api example](figures/graphql.png )
+
+* GraphQL API with authenthication: http://localhost:7000
+    * The default zendro username is *zendro-admin* and the password is *admin*.
+
+  ![api example](figures/login.png)
+  ![api example](figures/graphiql.png)
+
+
+If you wish to modify the default ports, adjust next files:
+> [Stop Zendro Instance](#stop), modify files and [start Zendro Instance](#start) again.
+
+* ./docker-compose-prod.yml
+* ./docker-compose-dev.yml
+* ./single-page-app/.env.production
+* ./single-page-app/.env.development
+* ./single-page-app/package.json
+* ./graphql-server/.env
+* ./graphiql-auth/.env.development
+* ./graphiql-auth/.env.production
+* ./graphiql-auth/package.json
+
+Also, if you wish to modify docker containers name or docker services names, adjust next files:
+> [Stop Zendro Instance](#stop), modify files and [start Zendro Instance](#start) again.
+
+* ./docker-compose-prod.yml
+* ./docker-compose-dev.yml
+
+Moreover, if you wish to modify keycloak IP adjust:
+* ipv4_address
+* subnet
+
+in *./docker-compose-prod.yml* and *./docker-compose-dev.yml*.
+
+
+#### Without docker
+If you prefer to use local setup with Keycloak, there are a few things to do after running Zendro:
+
+***MISSING***
+
+* install java
+* Install keycloak
+* Env vars and where to find them in keycloak, screenshots.
+* logs
+
```

Comment:
I guess something is missing in your doc here. the screenshots are weird for prerequisites.

Reply:
Yes, I meant that part is missing in the documentation.